sbi/inference/snpe/snpe_base.py

```diff
@@ -166,26 +163,22 @@ def __call__(
             # Run simulations for the round.
             theta, x, prior_mask = self._run_simulations(round_, num_sims)
```

Comment:
add check for `NaN`s (`handle_invalid_x`) here?

Reply:
Now I see that the `nn` utils that build the embeddings and flows check for invalid x themselves. Although this results in code repetition I can see the point that they want to do that - they might be used in other contexts where the input was not checked for `NaN`s. So I am actually fine with leaving this as it is.